rust/exchange_rate/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "exchange_rate"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+ic-cdk = "0.5.0"
+ic-cdk-macros = "0.5.0"
+ic-types ="0.3.0"
+serde = "1.0.126"
+serde_derive = "1.0.126"
+cronjob = "0.3.17"
```

Comment:
cronjob and tokio are not going to work.  getrandom and rand may not do what you want either depending on your requirements as they have no source of randomness, and indeed other than the call to management canister no way of getting any.  wasm-bindgen as well isn't going to work.

Reply:
Removed cronjob, wasm-bindgen and rand dependencies.  Thanks for the suggestions!